force-app/main/default/classes/SOQL.cls

```diff
@@ -117,7 +154,12 @@ public with sharing class SOQL {
      * @return A list of Case records.
      */
     public static List<Case> getFilterCaseOrigins() {
-        return null; // Replace null with the variable you used to store the result
+        List<Case> myCases = new List<Case>([
+            SELECT Id, Origin, Status, AccountId 
+            FROM Case 
+            WHERE Origin != null AND (Origin != 'Email' AND Origin != 'Web') WITH SECURITY_ENFORCED
```

Comment:
Use an empty string rather than `null` to also filter out blank values for Origin